addons/binding/org.openhab.binding.yeelight/ESH-INF/thing/channels.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
please apply the automatic code formatter (there should be tabs and no spaces here)

Reply:
You are right, copy and paste did not preserve the tabs apparently.

Reply:
Done.